azure-pipelines.yml

```diff
@@ -0,0 +1,106 @@
+variables:
+- group: 'Functions Host V1 Testing'
+- name: build_number
+  value: $[ counter('constant', 131400) ]
+- name: includeBuildNumberInVersion
+  value: ${{ 1 }}
+- name: configuration
+  value: Release
+- name: publish_path
+  value: bin
+- name: removeprefix_DOTNET_SKIP_FIRST_TIME_EXPERIENCE
```

Comment:
Curious what `removeprefix_` is for?

Reply:
Good question/catch. That's how the environment variable was defined in the AppVeyor build, but comparing it to our other builds, it looks like it shouldn't be there. Removing.